### PR TITLE
i18n(es): Update `i18n`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### Description

- What does this PR change? Give us a brief description.
Updated `i18n` based on these commits ce27486fabd3884ed4bca9372ebd72a0597ab765, 122b11aedcf03a784f32ac6919bd96f41a1d0d74 & a1fe5103c46c6c8b5e087d15437627ad1f3b2db2

### CODE REVIEW

File: docs/src/content/docs/es/guides/i18n.mdx [hunk 1]
```diff
@@ -140,9 +140,12 @@ Si no hay una traducción disponible para un idioma, Starlight mostrará a los l
 
 ## Traduce la UI de Starlight
 
+import LanguagesList from '../../../components/languages-list.astro';
```
Comment (HiDeoo) [inline docs/src/content/docs/es/guides/i18n.mdx:143]:
```suggestion
import LanguagesList from '../../../../components/languages-list.astro';
```

**Review by HiDeoo — DISMISSED** [review]
Thanks for the update.

I think the `<LanguageList/>` path needs to be updated for the build to work.

**Comment by dreyfus92** [comment]
Thank you so much for the review @HiDeoo 😁

**Comment by kevinzunigacuellar** [comment]
La Guitarra Toca Melodías ✅

**Review by delucis — APPROVED** [review]
Gracias a @dreyfus92 y @HiDeoo por la actualizacion y gracias a @kevinzunigacuellar por su guitarra.

**Pull request merged**